config/initializers/datadog.rb

```diff
@@ -0,0 +1,30 @@
+Datadog.configure do |c|
+  # Tracer
+  c.tracer hostname: ENV['DATADOG_HOST'],
+           port: ENV['DATADOG_PORT'],
+           tags: { app: ENV['PROJECT_NAME'] },
+           debug: false,
+           enabled: (ENV['DATADOG_TRACE'].to_s == 'true'),
+           env: Rails.env
+
+  # Rails
+  c.use :rails,
+        service_name: ENV['PROJECT_NAME'],
+        controller_service: "#{ENV['PROJECT_NAME']}-controller",
+        cache_service: "#{ENV['PROJECT_NAME']}-cache",
+        database_service: "#{ENV['PROJECT_NAME']}-db"
+
+  # Redis
+  c.use :redis, service_name: "#{ENV['PROJECT_NAME']}-redis"
+
+  # Sidekiq
+  c.use :sidekiq, service_name: "#{ENV['PROJECT_NAME']}-sidekiq"
+
+  # Net::HTTP
+  c.use :http, service_name: "#{ENV['PROJECT_NAME']}-http"
```

Comment:
this project doesn't use redis or sidekiq or net::http. Sure you want these?

Reply:
Yeah it won't hurt to have them in there, and it's a bit futureproof if they get added...

Reply:
eh. not likely.